src/components/atoms/ButtonNG/ButtonNG.tsx

```diff
@@ -51,6 +51,7 @@ export const ButtonNG: React.FC<ButtonProps> = ({
   iconOnly = false,
   iconName,
   iconSize = "1x",
+  ...extraProps
```

Comment:
`extraProps` in `ButtonNG` facilitate the use of `aria-*` as well as other props normally used with buttons